src/main/java/org/wildfly/security/audit/RotatingFileAuditEndpoint.java

```diff
@@ -68,12 +68,18 @@
         }
     }
 
+    /**
+     * Notice the following, this method is not thread safe and should be called inside a synchronized block
+     */
```

Comment:
Similar note would be better in superclass - FileAuditEndpoint - to define contract as suggested by @dmlloyd. To define that any implementation of this methods are called in synchronization block surrounding one log message processing.